spec.html

```diff
@@ -9556,15 +9572,13 @@ <h1>
         1. Let _lhs_ be ? ResolveBinding(_bindingId_, _environment_).
         1. Let _v_ be *undefined*.
         1. If _iteratorRecord_.[[Done]] is *false*, then
-          1. Let _next_ be Completion(IteratorStep(_iteratorRecord_)).
+          1. Let _next_ be Completion(IteratorStepValue(_iteratorRecord_)).
           1. If _next_ is an abrupt completion, set _iteratorRecord_.[[Done]] to *true*.
```

Comment:
is there a reason not to set [[Done]] inside IteratorStepValue?

Reply:
Oh, that's a nice simplification.

I guess the only reason I can think not to do so is that implementations might blindly copy it in cases where the value isn't actually going to get read. But I don't think we actually have to worry about that.